conf/bankDetails.routes

```diff
@@ -1,23 +1,23 @@
-GET         /what-you-need                                              @controllers.bankdetails.WhatYouNeedController.get
+GET         /what-you-need                                               @controllers.bankdetails.WhatYouNeedController.get
 
-GET         /accounts/:index                                             @controllers.bankdetails.BankAccountTypeController.get(index:Int, edit: Boolean ?= false)
-POST        /accounts/:index                                             @controllers.bankdetails.BankAccountTypeController.post(index:Int, edit: Boolean ?= false)
+GET         /account-type/:index                                             @controllers.bankdetails.BankAccountTypeController.get(index:Int, edit: Boolean ?= false)
```

Comment:
Does the indentation still work for these two?

Reply:
Fixed